centinel/command.py

```diff
@@ -73,10 +74,19 @@ def stop(self, timeout=None):
             return False
 
     def _invoke_cmd(self):
-        self.process = subprocess.Popen(self.command,
-                                        stdin=subprocess.PIPE,
-                                        stdout=subprocess.PIPE,
-                                        stderr=subprocess.STDOUT)
+        # if the command execution throws an exception,
+        # it should be caught and stored in a variable.
+        try:
+            self.process = subprocess.Popen(self.command,
+                                            stdin=subprocess.PIPE,
+                                            stdout=subprocess.PIPE,
+                                            stderr=subprocess.STDOUT)
+        except Exception as e:
```

Comment:
if there are other things to fix, then you should change this variable name; we decided that we aren't going to have 1 letter variable names